common/src/main/java/com/wynntils/functions/GuildFunctions.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.functions;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.functions.Function;
+import com.wynntils.core.consumers.functions.arguments.FunctionArguments;
+import com.wynntils.models.players.type.GuildRank;
+import com.wynntils.utils.type.CappedValue;
+import java.util.List;
+
+public class GuildFunctions {
+    public static class CappedGuildLevelPercentageFunction extends Function<CappedValue> {
+        @Override
+        public CappedValue getValue(FunctionArguments arguments) {
+            return new CappedValue(Models.Guild.getGuildLevelPercentage(), 100);
+        }
+
+        @Override
+        protected List<String> getAliases() {
+            return List.of("capped_guild_level_percentage");
+        }
+    }
+
+    public static class GuildLevelFunction extends Function<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            return Models.Guild.getGuildLevel();
+        }
+
+        @Override
+        protected List<String> getAliases() {
+            return List.of("guild_level");
+        }
+    }
+
+    public static class GuildLevelPercentageFunction extends Function<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            return Models.Guild.getGuildLevelPercentage();
+        }
+
+        @Override
+        protected List<String> getAliases() {
+            return List.of("guild_level_percentage");
+        }
+    }
```

Comment:
We don't really do these functions anymore as you can get the current level from the CappedValue function using `current(capped_guild_level_percentage))`